frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -746,13 +746,20 @@ export class TileViewPage extends React.Component {
     const { filterGroupNameMap } = this.props;
     const { filterGroupsShowAll } = this.state;
 
+    const maxShown = 5;
+    const showMoreText = i18n.t('public~{{numRemaining}} more', {
+      numRemaining: Object.keys(filterGroup).length - maxShown,
+    });
```

Comment:
Upon further thought -- I suspect we want `Show {{number}} more` instead of this one. It's a subtle difference but I think contextually makes more sense. Apologizes for my previous comments stating otherwise.